aiven/client/cli.py

```diff
@@ -31,9 +31,7 @@
 PLUGINS = []
 
 try:
-    from aiven.admin import (
-        plugin as adminplugin,
-    )  # pylint: disable=import-error,no-name-in-module
+    from aiven.admin import plugin as adminplugin  # pylint: disable=import-error,no-name-in-module
```

Comment:
Unrelated change?

Reply:
My setup runs `yapf` automatically. Should I revert it?

Reply:
Well, I think it's not that important, moreover, if it's a `yapf` related change. I'll merge this as is :)

Reply:
thanks!